### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by oriontvv**
This PR adds dependabot config

### CODE REVIEW

**Comment by jaymell** [comment]
Hi -- I think this is probably ok. The main reason I've been avoiding it is just b/c is 1) ignorance, 2) I don't want a lot of noise PRs that would cause us to inadvertently bump MSRV, but overall this is probably a good thing.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
```
Comment (jaymell) [inline .github/dependabot.yml:8]: I'm not clear -- what does package-ecosystem `docker` do here?

**Comment by oriontvv** [comment]
> that would cause us to inadvertently bump MSRV

We can ignore specific dependency if you wish so. Just tell the list of them.

File: .github/dependabot.yml (hunk 1 above)
Comment (jaymell) [reply .github/dependabot.yml:8]: Should we remove it then?

Comment (oriontvv) [reply .github/dependabot.yml]: done

**Review by jaymell — APPROVED** [review]
Alright, thanks! Let's see what happens :crossed_fingers:

**Comment by nihohit** [comment]
Thanks for the contribution!

**Pull request merged**